server/room.ts

```diff
@@ -548,7 +547,8 @@ export class Room {
       return;
     }
     redisCount('chatMessages');
-    const chatMsg = { id: socket.id, msg: data };
+    const user = this.roster.find((user) => user.id === socket.id);
+    const chatMsg = { id: socket.id, msg: data, isSubscriber: user?.isSubscriber };
```

Comment:
Do we need to do this on addChatMessage so actions like play/pause are also covered?